### PR TITLE
Create getting-started-participants.md

### PR DESCRIPTION
**Pull request by RachellCalhoun**

### CODE REVIEW

File: getting-started-participants.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- If you haven't yet, you probably want to start with going over the [Django Girls Tutorial](https://tutorial.djangogirls.org/en/) to get familiar with Django, Python, and Git. If you have some experience in one but not all of these, maybe skim through the parts you are familiar with. 
+
+**What is Git/Github?**  
+- If you are unfamiliar with Git and Github, [this tutorial](https://docs.github.com/en/get-started/quickstart/hello-world) might be helpful.  
+
+**What is Open Source Contributing?**   
+- Github has created a great [Contributing Guide](https://opensource.guide/how-to-contribute/) to go over what it's about, the process, and best practices. We'd encourage you to read it over. 
+
+**What are the program expectations?**    
+- You can read our [Navigator Guidelines](https://github.com/djangonaut-space/pilot-program/blob/main/navigators.md) to find more details about the responsibilities and expectations of our participants, or Navigators.
```
Comment (sarahboyce) [inline getting-started-participants.md:18]:
```suggestion
- You can read our [Djangonaut Guidelines](https://github.com/djangonaut-space/pilot-program/blob/main/djangonauts.md) to find more details about the responsibilities and expectations of our participants. 
```

**Review by sarahboyce — APPROVED** [review]
Looks good! 
Just want to update one link :+1:

**Review by tim-schilling — APPROVED** [review]
I'd agree that the links in the `What are the program expectations?` question could use some refinement, but I'm good with the rest of the file.

**Review by RachellCalhoun — COMMENTED** [review]
@tim-schilling is this better?Any other suggestions?

File: getting-started-participants.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+**What is Open Source Contributing?**
+- GitHub has created a great [Contributing Guide](https://opensource.guide/how-to-contribute/) to go over what it's about, the process, and best practices. We'd encourage you to read it over.
+
+**What are the expectations of participants?**
+- Have good standing throughout the program and uphold the [code of conduct](https://github.com/djangonaut-space/pilot-program/blob/main/CODE_OF_CONDUCT.md).
+- Attend weekly meetings to recieve feedback and support.
+- A commitment of 4 hours a week over the course of 3 months.
+- For more details about being a participant, please read our [Djangonaut Guidelines](https://github.com/djangonaut-space/pilot-program/blob/main/djangonauts.md).
+
```
Comment (RachellCalhoun) [inline getting-started-participants.md:22]: @tim-schilling  specifically here :)

Comment (tim-schilling) [reply getting-started-participants.md:22]: Looks great!

**Review by tim-schilling — APPROVED** [review]

**Pull request merged**